googleapis/google-cloud-node
# migrate code from googleapis/nodejs-service-usage

- chore: add initial files for launch
- feat: initial generation of library
- feat(samples): add example of listing service usage (#2)
- chore(deps): update dependency sinon to v11 (#5)
- feat!: bump to GA release level (#6)
- build(nodejs): use hyphen in package name (#7)
- chore: release 1.0.0 (#1)
- build: drop synth.metadata (#8)
- fix: GoogleAdsError missing using generator version after 1.3.0 (#9)
- feat: update serviceusage v1beta1 API (#10)
- chore: make generate_index_ts() deterministic (#11)
- chore: release 1.1.0 (#13)
- fix: make request optional in all cases (#19)
- fix(deps): google-gax v2.17.0 with mTLS (#27)
- chore: release 1.1.1 (#20)
- fix(deps): google-gax v2.17.1 (#29)
- chore: release 1.1.2 (#30)
- fix: failing tests for LRO check methods (#35)
- chore: release 1.1.3 (#36)
- fix(deps): google-gax v2.24.1 (#38)
- chore: release 1.1.4 (#39)
- feat: turns on self-signed JWT feature flag (#40)
- chore: release 1.2.0 (#41)
- chore: disable renovate dependency dashboard (#1194) (#42)
- fix(build): set default branch to main (#43)
- chore: release 1.2.1 (#44)
- build(node): update deps used during postprocessing (#1243) (#51)
- docs: Remove erroneous space in a docstring (#52)
- chore(cloud-rad): delete api-extractor config (#53)
- chore: fix the wrong post processor image (#55)
- chore(deps): update dependency @types/node to v16 (#54)
- chore(deps): update dependency sinon to v12 (#56)
- docs(samples): add example tags to generated samples (#57)
- build: add generated samples to .eslintignore (#58)
- docs(node): support "stable"/"preview" release level (#1312) (#61)
- chore: add api_shortname and library_type to repo metadata (#60)
- docs(badges): tweak badge to use new preview/stable language (#1314) (#63)
- test(nodejs): remove 15 add 16 (#1322) (#65)
- build: update copyright year to 2022 (#68)
- chore(deps): update dependency sinon to v13 (#72)
- docs(nodejs): version support policy edits (#1346) (#74)
- docs(samples): include metadata file, add exclusions for samples to handwritten libraries (#75)
- chore: update v2.14.2 gapic-generator-typescript (#79)
- chore: fix api_shortname in repo-metadata (#80)
- build(node): update client library version in samples metadata (#1356) (#87)
- chore(deps): update dependency @types/mocha to v9 (#88)
- chore(deps): update dependency sinon to v14 (#91)
- build!: update library to use Node 12 (#92)
- chore(main): release 2.0.0 (#94)
- chore(deps): update dependency jsdoc-region-tag to v2 (#97)
- chore(deps): update dependency jsdoc-fresh to v2 (#96)
- feat: support regapic LRO (#98)
- chore(main): release 2.1.0 (#99)
- chore(deps): update dependency linkinator to v4 (#103)
- chore: remove unused proto imports (#104)
- fix: better support for fallback mode (#105)
- fix: change import long to require (#106)
- chore(main): release 2.1.1 (#107)
- fix: do not import the whole google-gax from proto JS (#1553) (#109)
- fix: allow passing gax instance to client constructor (#110)
- fix: preserve default values in x-goog-request-params header (#112)
- test: use fully qualified request type name in tests (#114)
- fix(deps): use google-gax v3.5.2 (#118)
- chore(deps): update dependency @types/node to v18 (#116)
- chore(deps): update dependency jsdoc to v4 (#120)
- fix: regenerated protos JS and TS definitions (#121)
- chore(main): release 2.1.2 (#111)
- build: add release-please config, fix owlbot-config

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

Fixes #<issue_number_goes_here> 🦕
